app/js/main.js

```diff
@@ -201,6 +211,39 @@ function setupMeteringDemo(subscriptions) {
     location.reload();
   });
 
+  // Determine whether there is a publisher based entitlement
+  let entitlement = null;
+  switch (getQueryParams().publisherEntitlement) {
+    case 'free':
+      entitlement = EntitlementType.FREE;
+      break;
+    case 'subscribed':
+      entitlement = EntitlementType.SUBSCRIBED;
+      break;
+    case 'metered':
+      entitlement = EntitlementType.METERED;
+      break;
+    default:
+      // See if someone set publisherEntitlement to one of the enums
```

Comment:
The comment doesn't seem to match the code below it. From the comment, I would assume the code was checking if a URL param matches enum values, but the code is just accepting any value from the URL param